libs/llm/providers/ollama/adapters/tuda-llama2/tuda.llama2.prompts.ts

```diff
@@ -1,7 +1,18 @@
-export const llama2Prompt = `Given is the following task-oriented dialog between a human user (<user>) and a virtual agent (<assistant>). 
+import { PromptTemplate } from 'libs/llm/prompt/prompt.template';
+
+export const llama2chatPrompt = PromptTemplate.create(
+  'chat',
+  `
+Given is the following task-oriented dialog between a human user (<user>) and a virtual agent (<assistant>). 
 Previously, this conversation went wrong because the virtual agent made a statement that was contextually incorrect.
 The human user reacted accordingly (No, I don't think so. I asked for assistance with my legal dispute.). 
```

Comment:
The example "No, I don't think so. I asked for assistance with my legal dispute." seems very specific. Is it correct?

Reply:
Yes this is a template for a fine-tuned model developed in SERMAS